src/globals.h

```diff
@@ -77,7 +77,7 @@ XTRN int	SyslogOutput;
 XTRN time_t	StartTime;
 XTRN int	NoFork;
 XTRN int        PermitAnyCrontab;
-XTRN char       MailCmd[MAX_COMMAND];
+XTRN char       MailCmd[MAX_COMMAND+1]; // +1 for terminator
```

Comment:
Please do not use // as comment separator. Use /* ... */

Reply:
Also please note there is strncpy(mailcmd, MailCmd, MAX_COMMAND); in do_command.c which is missing termination with '\0' in case the MailCmd string is of length MAX_COMMAND.